syntax = "proto3";

service CompetitionTool {
  rpc Initialize (stream SDCTestCase) returns (InitializationReply) {}

// bidirectional streaming for high flexibility
  rpc Select (stream SDCTestCase) returns (stream SelectionReply) {}
}

message SDCTestCase {

}

message InitializationReply {

}


message SelectionReply {
  string testId = 1;
}
